unit_test/sparse/Test_Sparse.hpp

```diff
@@ -18,6 +18,7 @@
 //#include "Test_Sparse_spmv_bsr.hpp"
 #include "Test_Sparse_sptrsv.hpp"
 #include "Test_Sparse_trsv.hpp"
+#include "Test_Sparse_Conversion.hpp"
```

Comment:
Should we split this to keep compile times low?

Reply:
Thanks for your feedback, @srajama1. I refactored the code to split conversion routines into their own hpp files. I also moved CscMat to KokkosKernels_TestUtils.hpp and added getters for getting the val, row_id, and col_map views.

I will start implementing csc2csr now.